101-vm-full-disk-encrypted-rhel/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "value": "GEN-USER"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "aadClientID": {
```

Comment:
null the param values since CI won't handle this and user must supply something specific